src/transformState.js

```diff
@@ -6,6 +6,31 @@
  */
 function transformState(state, actions) {
   // write code here
+  const type = 'type';
+  const extraData = 'extraData';
+  const keysToRemove = 'keysToRemove';
```

Comment:
It's better to use destructuring instead of creating some string variables, if some fields will change - your code won't work